### ISSUE TITLE
TruncatedText

### ISSUE THREAD

**Issue by fabriciofx** [yegor256/cactoos#196]
Let's create `TruncatedText`, which will do what StringUtils.truncated() in [Apache Commons](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#truncate-java.lang.String-int-int-) doing:
```java
String truncated = new TruncatedText("hello world", 8).asString();
```
And `truncated` will be equal to `hello wo`

**Comment by 0crat** [comment]
@fabriciofx I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

**Comment by fabriciofx** [comment]
Closed this issue in favor of SubText (issue #194 ).
